.travis.yml

```diff
@@ -2,9 +2,10 @@ language: node_js
 notifications:
   email: true
 node_js:
-- '8'
+  - '8'
+install: travis_wait 30
```

Comment:
@sohkai Is this the correct place for `travis_wait` flag? I want to extend the log output build error, was not sure about the specific place 🤔

Reply:
`travis_wait` acts like a decorator, so you'll still need to provide the actual command to be run here (`npm install`).

However, I noticed there's only the `prepublishOnly` script defined, so I think the slow part is actually on `npm publish` as part of the deploy. Not sure if there's a way to tell travis not to timeout on that, but we may need to modify the scripts a bit so that we can run `npm run start` with the timeout and publish afterwards.